js/element-outerhtml/demos/index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Работа Element.outerHTML — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto:wght@400;500&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
```

Comment:
```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto:wght@400;500&display=swap">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
```

На странице нигде не используется Roboto Mono, хотя аж дважды подключается в шапке.

Reply:
Шрифт используется в css. Но двойное подключение это некруто, спасибо что заметил.

Reply:
В стилях используется только Roboto, это другое семейство.

<img width="414" alt="image" src="https://user-images.githubusercontent.com/105274/126147842-f46db5be-86b2-4e1f-9135-2a3b2bfa19b7.png">

Reply:
Точно, обновил ссылку на шрифт